### PR TITLE
chore: move from Travis to GitHub Actions

### PR DESCRIPTION
**Pull request by Trott**
Travis isn't running because we're out of credits. Move to GitHub
Actions.

### CODE REVIEW

**Comment by Trott** [comment]
I've disabled Travis for the repo so if we want it back, it will need to be re-enabled.

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
```
Comment (richardlau) [inline .github/workflows/node.js.yml:24]: I think I've seen other repos move to setup-node@v2.

Comment (Trott) [reply .github/workflows/node.js.yml]: OK, I `s/1/2/` and pushed a fixup commit.

**Review by richardlau — APPROVED** [review]
This LGTM. GitHub hasn't run this because this has come from a fork.

**Review by richardlau — APPROVED** [review]

**Pull request merged**